plugins/techdocs-module-addons-contrib/api-report.md

```diff
@@ -0,0 +1,33 @@
+## API Report File for "@backstage/plugin-techdocs-module-addons-contrib"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { ComponentType } from 'react';
+
+// @public
+export const ReportIssue: ComponentType<ReportIssueProps>;
+
+// @public (undocumented)
+export type ReportIssueProps = {
+  debounceTime?: number;
+  templateBuilder?: ReportIssueTemplateBuilder;
+};
+
+// @public (undocumented)
+export type ReportIssueTemplate = {
+  title: string;
+  body: string;
+};
+
+// @public (undocumented)
+export type ReportIssueTemplateBuilder = (
+  selection: Selection,
```

Comment:
Wondering whether `Selection` is the best parameter here? Thinking about alternatives like for example the text content. Another concern is forwards compatibility, perhaps consider wrapping this in an object so that you can pass additional values in the future without needing to add more parameters.